scripts/migrate_piwik_historical_data.py

```diff
@@ -0,0 +1,152 @@
+import MySQLdb
+from socket import inet_ntoa
+from binascii import b2a_hex
+import pprint
+from keen import KeenClient
+import urlparse
+
+def main():
+
+    keen_client = KeenClient(
+        project_id='56abc1c759949a6734c3f32b',
+        write_key='248de5ffaf1c829bf74dc6e4e0fc3af6beaa7f5194c2d163699213b66e8cf1fbc41a1adc1dd9b6ec3681ac31584bd0c9847c8d9ca223e44e99e60af219be19c55a9261437020069f92c5825cc2995cba57976ed485e3a9f314e3a1d21d032e53'
+    )
+
+    try:
+        db = MySQLdb.connect(host='127.0.0.1', port=8889, user='root', passwd='root', db='piwikOrig')
```

Comment:
These values should not be hardcoded. Maybe put this script in `scripts.keen` and add a `settings` folder where this info can be configured. See `scripts.osfstorage` for an example.